.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+
+Thank you for filing a bug report. :tada::+1:
+
+Please read this and fill out this template. If unsure about something, just do as best as you're able.
+
+Remember that this is a volunteer-driven project, and that contributions are welcome.
+
+Please note that this is only for reporting problems, not for asking for support. Please use the Django Mailing list or StackOverflow to ask for help.
+
+Please check that no-one else has reported this bug before by searching existing issues.
+
+If you are reporting a SECURITY issue, please email hello@jambonsw.com instead.
+
+If you are requesting a feature, please refer to the documentation to make sure it is consistent with the package's goals.
+
+To Use this Template:
+
+* Use a clear and descriptive title for the issue to identify the problem
+* Fill out what you can
+* Delete what you do not fill out
+-->
+
```

Comment:
This may not be a problem, but when I open up the page using "View", the Issue template starts with "Specification" and all the sentences before that are not shown. Further, only the title headings are shown, not any of the sentences defining those headings.
(These do look like HTML comments, so that might be OK).